index.js

```diff
@@ -0,0 +1,51 @@
+function solveEquation(ans) {
+  switch (ans[1]) {
+    case '+':
+      return ans[0] + ans[2]
+    case '-':
+      return ans[0] - ans[2]
+    case '*':
+      return ans[0] * ans[2]
+    case '/':
+      return ans[0] / ans[2]
+    default: return NaN
+  }
+}
+
+function isWorkingValue(value) {
+  const operators = ['+', '*', '-']
+
+  return isNumber(value) || operators.includes(value)
+}
+
+function isNumber(value) {
+  return !isNaN(value)
+}
+
+function isIgnoredValue(value) {
+  return value === ''
+}
```

Comment:
I would call this in your isWorkingValue function. Then it is solely responsible for determining a good value and you don't need to split that between it and calculate which is really what is happening now since calculate is calling both.